packages/components/src/SpectrumUtils.ts

```diff
@@ -0,0 +1,54 @@
+import { defaultTheme } from '@adobe/react-spectrum';
```

Comment:
If all we're doing is using the `defaultTheme`, I'd rather just add `@react-spectrum/default-theme` as a dependency and get rid of the peer dependency/breaking change.
We're using the ACL Editor as a testing ground for trying out the spectrum components, I don't think the first step should be requiring consumers of our `@deephaven/components` package to add spectrum as a dependency as well.

Reply:
I've made the change